front/app/hooks/useIdeaMarkers.ts

```diff
@@ -6,10 +6,16 @@ import { ideaDefaultSortMethodFallback } from 'services/participationContexts';
 
 interface Props {
   phaseId?: string | null;
-  projectIds?: string[] | null;
+  // As long as we JSON.parse this value, it can't be
```

Comment:
These comments can go if we aren't stringifying/parsing the JSON anymore

Reply:
We're still parsing in this hook. It's a hook unrelated to our work. Just wanted to avoid possible silent bugs in the future.

Reply:
Got it. Don't need to change it here (we can discuss the linter rule at the CDM) but we could avoid the potential for the parsing bug by just using the actual `projectIds` object in the query params object, and the stringified version (with linter override) just for the dependency array